index.html

```diff
@@ -187,6 +187,17 @@ <h3 id="registry">
         </dd>
       </dl>
     </section>
+    <section id="conformance">
+      <p>
+        The working group will demonstrate implementation experience by
+        creating a <a href="https://w3c-test.org/payment-method-id/">test
+        suite</a> and having at least two independent implementations pass each
+        mandatory test (i.e., each test the corresponds to a MUST requirements
```

Comment:
mandatory test (i.e., each test the corresponds to a MUST requirements

_requirement_ (without the _s_)